.github/workflow/deploy.yml

```
@@ -0,0 +1,36 @@
+name: deploy-book
+
+# Only run this when the master branch changes
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+# This job installs dependencies, builds the book, and pushes it to `gh-pages`
+jobs:
+  deploy-book:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    # Install dependencies
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        conda install -f environment.yaml
```

```suggestion
        conda env create -f environment.yaml
```

I gave you incorrect information earlier, this is the correct command :) https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file

---

Further additions to this command I got from this blog post https://autobencoder.com/2020-08-24-conda-actions/

---

Thanks for sharing the correct command - the blog post is also a great one!!!